config/config.go

```diff
@@ -44,9 +44,12 @@ var (
 
 // TelemetryOpts represents the options for configuring telemetry paths on local filesystem.
 type TelemetryOpts struct {
-	RootPath            string `help:"define Percona telemetry root path on local filesystem." env:"PERCONA_TELEMETRY_ROOT_PATH" default:"/usr/local/percona/telemetry"`
-	PSMetricsPath       string `kong:"-"`
-	PSMDBMetricsPath    string `kong:"-"`
+	RootPath      string `help:"define Percona telemetry root path on local filesystem." env:"PERCONA_TELEMETRY_ROOT_PATH" default:"/usr/local/percona/telemetry"`
+	PSMetricsPath string `kong:"-"`
+	// For PSMDB (mongod) component
+	PSMDBMetricsPath string `kong:"-"`
+	// For PSMDB (mongos) component
+	PSMDBSMetricsPath   string `kong:"-"`
```

Comment:
TBH it's not so easy to distinguish these two names at the first sight.
How about `PSMDBMongodPath` and `PSMDBMongosPath` ?

Reply:
agree, will adjust